atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -65,7 +65,7 @@ public static void assertAndLog(boolean cheapTest, String format, Object... args
 
     public static void assertAndLogWithException(boolean cheapTest, String msg, Throwable t) {
         if (!cheapTest) {
-            LoggerFactory.getLogger(AssertUtils.class).error(msg, t);
+            LoggerFactory.getLogger(AssertUtils.class).error("Assertion {} with exception ", msg, t);
             assert false : msg;
```

Comment:
Don't think we run with assertions turned on so this does nothing.  Don't think we should remove it now though.